edx-platform/pearson-theme/lms/templates/header/navbar-authenticated.html

```diff
@@ -36,6 +37,11 @@
           ${_("Profile")}
         </a>
       </div>
+    % endif
+     % if show_catalogue_link:
```

Comment:
@amalbas Just a extra indentation here, needs to be at the same level of the % endif declaration.